src/lfs_config.h

```diff
@@ -209,19 +213,25 @@ uint32_t lfs_crc(uint32_t crc, const void *buffer, size_t size);
 // Allocate memory, only used if buffers are not provided to littlefs
 // Note, memory must be 64-bit aligned
 static inline void *lfs_malloc(size_t size) {
-#ifndef LFS_NO_MALLOC
-    return heap_caps_malloc(size, MALLOC_CAP_8BIT | MALLOC_CAP_INTERNAL);
-#else
+#if defined(CONFIG_LITTLEFS_MALLOC_STRATEGY_GENERAL)
+    return malloc(size);
+#elif defined(CONFIG_LITTLEFS_MALLOC_STRATEGY_INTERNAL)
+    return heap_caps_aligned_alloc(8, size, MALLOC_CAP_8BIT | MALLOC_CAP_INTERNAL);
+#elif defined(CONFIG_LITTLEFS_MALLOC_STRATEGY_SPIRAM)
+    return heap_caps_aligned_alloc(8, size, MALLOC_CAP_8BIT | MALLOC_CAP_SPIRAM);
```

Comment:
per my review comment, I **think** these can really be 4 instead of 8. Regardless, it doesn't seem like a big deal.

Reply:
Fixed